Assets/Scripts/Circuit/Triggerable/Door.cs

```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+
+public class Door : TriggerableAbstract
+{
+    public GameObject ClosedDoor;
+    public GameObject OpenDoor;
+
+    private void Awake()
```

Comment:
if closed and open are distinct states, why cant we just do:

```
private void Awake() {
ClosedDoor.SetActive(IsOn);
IsOn = false;
}
```

or, maintaining gameobj check


```
private void Awake() {
ClosedDoor.SetActive(ClosedDoor != null);
IsOn = false;
}
```

Reply:
(similarly for all ClosedDoor != null, OpenDoor !=null